### ISSUE TITLE
[Sprint 1 5/23/20] Scrum Updates

### ISSUE THREAD

**Issue by yehee** [CPSC436/ThinkSmall#1]
- What did you work on this past iteration (2 weeks)?
- What were any major issues/challenges you ran into?
- What do you plan to work on for this coming iteration (2 weeks)?

**Comment by johnskimca** [comment]
John Kim - CSID: q6i8

 - In this past iteration, our team worked on researching a suitable webApp idea. My personal tasks included the following, researching Job Search APIs to determine the feasibility of a project centered around the concept of a unified job board sourced from several different websites. I researched the APIs of Linkedin, Glassdoor, indeed, Github and Stackoverflow.

- The key challenge I ran into was trying to decipher how to use many of the APIs of the companies I listed above. Some companies would outright reject non-partners using their APIs. Others would provide some information and functionalities that were confusing to say the least. Only one company, indeed, was feasible but still required a publisher account approved by the company. 

- In the next two weeks, I will communicate with the team members who constructed the prototypes to ensure that my contribution to the visual aspect of the project is in line with their vision.

**Comment by OriginalJangster** [comment]
Yuree Jang (CSID: n6m2b)

- I worked on searching up APIs that could work for our small business web app, particularly the Yelp API, Shopify API, Google API, Uber Eats API, and also web scraping scripts (ParseHub). Once we decided on our app concept, I wrote the product description for the project progress report, as well as helped to determine the minimum, standard, and stretch requirements of the same report.

- One of the major issues was trying to figure out what direction this project to go, since we first thought it should be an aggregated directory and shopping app with all the products from all the small businesses in Vancouver, sorted/ filtered by category. However, this idea had a technical blockage, but we instead used this to change the direction of the project into something more relevant and useful for the current times: an app to help small businesses by connecting them to volunteers in the community that can help them with their needs.

- Once I have completed Assignment 2 and understand React and Redux properly, I want to implement at least one of the components that our app will need.

**Comment by anhyesun** [comment]
DaeBak_Sprint_2020_01
Scrum Update: Hyesun An (c5g2b)

**1. During this sprint, I**  
* Suggested project ideas and expanded the selected idea with a research
* Scratched view of prototyping based on the project minimum requirements  

**2. My Plans for the next sprint is**
* Developing at least one component other than App.js/main item.js
* Adding at least one piece of styling in prototypes with CSS

**3. Challenges I had** 
* Combined good ideas from the brainstorming
* Searched appropriate API that matched with project requirements

**Comment by yehee** [comment]
Alice Kim (d7q0b)
- In this past iteration, I've investigated related API's and worked on prototyping for potential project ideas to better understand its feasibility.
- We had several potential options for our project, so we narrowed down into two ideas and started working on prototyping and API investigation so that we have more information before we make a solid decision on what to go with.
- I'm planning to work on getting the repo setup, and hash out the details to get started on with the project.
